gen3/bin/kube-setup-secrets.sh

```diff
@@ -206,36 +206,48 @@ if [[ -f "$(gen3_secrets_folder)/creds.json" ]]; then # update fence secrets
   if ! g3kubectl get secrets/fence-config > /dev/null 2>&1; then
     # load updated fence-config.yaml into secret if it exists
     fence_config=$(gen3_secrets_folder)/apis_configs/fence-config.yaml
-    if [[ -f ${fence_config} ]]; then
-      echo "loading fence config from file..."
-      if g3kubectl get secrets/fence-config > /dev/null 2>&1; then
-        g3kubectl delete secret fence-config
-      fi
+    fence_pub_config=$(gen3_secrets_folder)/apis_configs/fence-config-public.yaml
+    fence_sec_config=$(gen3_secrets_folder)/apis_configs/fence-config-secret.json
+    if [[ -f ${fence_pub_config} && -f ${fence_sec_config} ]]; then
```

Comment:
The fence secret config is $(gen3_secrets_folder)/apis_configs/fence-config.yaml - everything under the secrets folder is secret.

The logic should be something like:
* secret part == $(gen3_secrets_fodler)/apis_configs/fence-config.yaml
* public part == $(g3k_manifest_folder)/fence/fence-config-public.yaml if it exists, otherwise empty string
* merge the 2 into a temporary file
* make a secret out of the temporary file
* erase the temporary file
something like that ...
you probably want to put that logic into `bin/secrets.sh`, and remove it from `kube-setup-secrets`